sound/mpc/Makefile

```diff
@@ -6,21 +6,24 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mpc
-PKG_VERSION:=0.30
-PKG_RELEASE:=3
+PKG_VERSION:=0.32
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://www.musicpd.org/download/mpc/0
-PKG_HASH:=65fc5b0a8430efe9acbe6e261127960682764b20ab994676371bdc797d867fce
+PKG_HASH:=7961d95b7ce019996beab281cf957e905667c989c53fffd13ade5e62fea331c7
 
 PKG_LICENSE:=GPL-2.0-or-later
 PKG_LICENSE_FILES:=COPYING
 PKG_MAINTAINER:=Ted Hess <thess@kitschensync.net>
 
-PKG_FIXUP:=autoreconf
+PKG_BUILD_DEPENDS:=meson/host
 PKG_INSTALL:=1
```

Comment:
PKG_BUILD_PARALLEL could be good here. I don't know if Meson has problems with it.

Reply:
It does builds via ninja in parallel, just doesn't require PKG_BUILD_PARALLEL, that's a gnu make setting that's not required with meson.